semgrep_output_v1.atd

```diff
@@ -1113,29 +1114,35 @@ type scan_request = {
 }
 
 (* Response by the backend to the CLI to the POST /scans *)
-(* TODO
 type scan_response = {
-    scan: scan_response_bis;
+    scan: scan_info;
+    scan_config: scan_configuration;
+    engine_config: engine_configuration;
 }
-type scan_response_bis = {
-    (* the scan id *)
-    id: string;
-    (* since 1.37.0 *)
-    ?enabled_products: product list;
+
+(* meta info about the scan *)
+type scan_info = {
```

Comment:
What about splitting this into separate types: `scan_id` and `scan_deployment`? Then the scan_response can simply have 
```
id: scan_id;
deployment: scan_deployment;
```

Reply:
Yeah I suppose you're right. Having this type defined means we can add additional scan info metadata later if needed.